### DISCUSSION TITLE
recorded what it looks like to have 50 people playing at once!

### DISCUSSION THREAD

**Discussion by RDude-RampageTeam** [efroemling/ballistica#476] [Show and tell]
do i have your respect now froemling???

https://user-images.githubusercontent.com/41275888/179612207-8e7f425f-5f07-4b26-ac0b-93e833abb3ca.mp4

**Comment by EraOSBeta** [comment]
Nice :+1: Also you could make it more realistic if you make each spaz react to each input differently (eg. pressing the bomb button will call the bomb button on one spaz and the punch button on another spaz, or going to the left is going to the right for some of the spazes)
